src/components/PreferencesPage.vue

```diff
@@ -497,7 +502,18 @@ export default {
             this.selectedAuthInstance = this.getPreferenceString("auth_instance_url", this.selectedInstance);
 
             this.sponsorBlock = this.getPreferenceBoolean("sponsorblock", true);
-            if (localStorage.getItem("selectedSkip") !== null) {
+            if (localStorage.getItem("skipOptions") !== null) {
+                const skipOptions = JSON.parse(localStorage.getItem("skipOptions"));
+                if (skipOptions.sponsor !== undefined) this.skipSponsor = skipOptions.sponsor;
+                if (skipOptions.intro !== undefined) this.skipIntro = skipOptions.intro;
+                if (skipOptions.outro !== undefined) this.skipOutro = skipOptions.outro;
+                if (skipOptions.preview !== undefined) this.skipPreview = skipOptions.preview;
+                if (skipOptions.interaction !== undefined) this.skipInteraction = skipOptions.interaction;
+                if (skipOptions.selfpromo !== undefined) this.skipSelfPromo = skipOptions.selfpromo;
+                if (skipOptions.music_offtopic !== undefined) this.skipMusicOffTopic = skipOptions.music_offtopic;
+                if (skipOptions.poi_highlight !== undefined) this.skipHighlight = skipOptions.poi_highlight;
+                if (skipOptions.filler !== undefined) this.skipFiller = skipOptions.filler;
+            } else if (localStorage.getItem("selectedSkip") !== null) {
```

Comment:
Please use `getPreferenceString` here.

Reply:
Sure thing, this is based on existing code and I wasn't sure why there were two different ways to read preferences.